spec/index.bs

```diff
@@ -16732,6 +16733,32 @@ expose real values whenever the feature is available on the adapter:
 - New WGSL extensions:
     - [=extension/subgroups=]
 
+<h3 id=storage-texture-format-tier1 data-dfn-type=enum-value data-dfn-for=GPUFeatureName>`"storage-texture-format-tier1"`
+</h3>
+
+Allows the {{GPUStorageTextureAccess/"read-write"}} {{GPUStorageTextureAccess}} on the textures with
+one of the below {{GPUTextureFormat}}s:
+- {{GPUTextureFormat/"rgba32float"}}
+- {{GPUTextureFormat/"rgba32uint"}}
+- {{GPUTextureFormat/"rgba32sint"}}
+- {{GPUTextureFormat/"rgba16float"}}
+- {{GPUTextureFormat/"rgba16uint"}}
+- {{GPUTextureFormat/"rgba16sint"}}
+- {{GPUTextureFormat/"rgba8unorm"}}
+- {{GPUTextureFormat/"rgba8uint"}}
+- {{GPUTextureFormat/"rgba8sint"}}
+
+Allows the {{GPUTextureUsage/STORAGE_BINDING}} usage with all {{GPUStorageTextureAccess}}es on the
+textures with one of the below {{GPUTextureFormat}}s:
+- {{GPUTextureFormat/"r16float"}}
+- {{GPUTextureFormat/"r16uint"}}
+- {{GPUTextureFormat/"r16sint"}}
+- {{GPUTextureFormat/"r8unorm"}}
+- {{GPUTextureFormat/"r8uint"}}
+- {{GPUTextureFormat/"r8sint"}}
+
+This feature adds no [=optional API surfaces=].
+
 # Appendices # {#appendices}
 
 ## Texture Format Capabilities ## {#texture-format-caps}
```

Comment:
I haven't reviewed the tables to make sure they match the description. I or someone will need to double check that once we're satisfied with the description.